tests/pds/dataservice_test.go

```diff
@@ -396,6 +396,109 @@ var _ = Describe("{ScaleUPDataServices}", func() {
 	})
 })
 
+var _ = Describe("{RunIndependentAppNonPdsNS}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RunIndependentAppNonPdsNS", "Runs an independent app on a non-PDS namespace and then enables PDS on this namespace", nil, 0)
+	})
+	ns := ""
+	result := false
+	var podName string
+
+	It("Create an independent app in a non PDS namespace and then enable PDS on this namespace", func() {
+		Step("Create a temporary namespace on the cluster for creating an independent app", func() {
+			ns, result = pdslib.CreateTempNS()
+			if !result {
+				dash.VerifyFatal(result, true, "Failure in creating namespace on the target cluster. Exiting the Test case with failure")
+			}
+			log.InfoD("Namespace %s created for creating a Non-PDS App", ns)
+		})
+		Step("Create an Independent app in a non-PDS namespace", func() {
+			if pdslib.CreateIndependentPV("mysql-pv") {
```

Comment:
where are we using the pv created here ? If pv creation is not required kindly remove it

Reply:
As discussed - this is a manual pv